Dockerfile

```diff
@@ -1,8 +1,9 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi9/go-toolset:1.22.9 AS builder
+FROM docker.io/golang:1.23 AS builder
```

Comment:
This will likely make productization harder later. Why is it necessary?

We may want to check when is the new go toolset being released.

Reply:
Some of our dependencies require go 1.23. I need to hunt them down.